distro/runtime/concierge/smarthome.xargs

```diff
@@ -64,6 +65,7 @@
    javax.security.auth.spi,\ 
    javax.security.auth.x500,\
    javax.security.cert,\
+   javax.sound.sampled,\
```

Comment:
javax.sound is NOT part of CP2/CP3.
I would hope that it also isn't needed, unless you want to include org.eclipse.io.javasound (which imho can be left away).